themes/custom/droopler_theme/scss/components/_main-menu.scss

```diff
@@ -0,0 +1,692 @@
+/*
```

Comment:
no wiec, cały ten plik, to sie nie powinien znaleźć w Twoim PR,
on został rozbity na dwa, w osobnym katalogu, sprawdz: 
![image](https://user-images.githubusercontent.com/37533838/72022914-df768080-3271-11ea-8587-0c812734892d.png)
tam wcześniej robiłeś jakieś zmiany, więc musisz obczaic kody które zmieniałeś i je odnaleźć w którymś z nowych pliczkow